projects/js-packages/charts/src/components/private/radial-wipe-animation/index.ts

```diff
@@ -0,0 +1,2 @@
+export type { RadialWipeProps as RadialWipeAnimationProps } from './radial-wipe-animation';
+export { default as RadialWipeAnimation } from './radial-wipe-animation';
```

Comment:
Maybe rename these to match the filename and make the export cleaner?
```suggestion
export {
	default as RadialWipeAnimation,
	type RadialWipeAnimationProps,
} from './radial-wipe-animation.tsx';

```